freqtrade/plugins/pairlist/RemotePairList.py

```diff
@@ -236,15 +237,15 @@ def gen_pairlist(self, tickers: Tickers) -> List[str]:
 
                 if file_path.exists():
                     with file_path.open() as json_file:
-                        # Load the JSON data into a dictionary
-                        jsonparse = rapidjson.load(json_file, parse_mode=CONFIG_PARSE_MODE)
-
                         try:
+                            # Load the JSON data into a dictionary
+                            jsonparse = rapidjson.load(json_file, parse_mode=CONFIG_PARSE_MODE)
                             pairlist = self.process_json(jsonparse)
                         except Exception as e:
                             if self._init_done:
                                 pairlist = self.return_last_pairlist()
                                 logger.warning(f'Error while processing JSON data: {type(e)}')
+                                logger.debug(f'Error while processing JSON data: {e}')
```

Comment:
what's the difference / expectation to log both warning and debug ... ? 
if you're after the reason (like, stacktrace) - you should enable stacktrace instead (or log at exception level, where this is auto-enabled) ... ?